src/app/components/applications/SSV/models/index.ts

```diff
@@ -0,0 +1 @@
+export { KeyShareItem, Validator } from './registerValidator.models';
```

Comment:
absolute

Reply:
you don't use absolute path in index files